README.md

```diff
@@ -1,27 +1,87 @@
+| `leasetime` | integer |                          | Request a specific lease time in seconds.                                          |     N    |
+| `metric`    | integer |                          | Metrics are used to prefer an interface over another one, lowest wins.             |     N    |
+
+
+### port
+| Option | Type   | Default | Description                     | Required |
+|--------|--------|---------|---------------------------------|:--------:|
+| port   | string |         | Interface name to add to bridge |     Y    |
 
-## Example Playbook
-### Playbook:
 
-### Vars:
-```yml
-```
-### Result:
+### interfaces
+| Option        | Type            | Default                                         | Description                                                                               |             Required            |
+|---------------|-----------------|-------------------------------------------------|-------------------------------------------------------------------------------------------|:-------------------------------:|
+| `name`        | string          |                                                 | Name of the interface to configure, ignored if `mac` is specified                         |  only if `mac` is not specified |
+| `mac`         | string          |                                                 | MAC-Address of the interface to configure. If `name` is specified, `mac` is autodetected. | only if `name` is not specified |
+| `bring_up`    | boolean         | `{{ network_management_default_bring_up }}`     | Automatically bring interface link up                                                     |                N                |
+| `type`        | [string](#type) | `{{ network_management_default_type }}`         | Specify how the network interface should be configured                                    |                N                |
+| `dhcp`        | [dict](#dhcp)   | `{{ network_management_default_dhcp_options }}` | Additional options for dhcp interfaces                                                    |                N                |
+| `nameservers` | list of strings | `{{ network_management_nameservers }}`          | Nameservers to use                                                                        |                N                |
+| `mtu`         | integer         |                                                 | MTU size                                                                                  |                N                |
+| `ip`          | string          |                                                 | IP address to assign as CIDR                                                              |                N                |
+| `ips`         | list of strings |                                                 | IP addresses to assign to interface as CIDR                                               |    only if `type` is `static`   |
```

Comment:
IP addresses to assign (IP address with CIDR suffix)

Reply:
fixed